axelor-supplychain/src/main/java/com/axelor/apps/supplychain/service/invoice/generator/InvoiceGeneratorSupplyChain.java

```diff
@@ -129,4 +143,54 @@ protected Invoice createInvoiceHeader() throws AxelorException {
 
     return invoice;
   }
+
+  @Override
+  public void computeInvoice(Invoice invoice) throws AxelorException {
```

Comment:
Instead of overriding this method you can add necessary changes in its super method